I1724 ibm style guide (#1221)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/proc_choosing-the-sidecar-image.adoc

* eclipse/che-remote-plugin-openshift-connector-0.0.21
====

For a VS Code extension with external dependencies not found in one of the ready-to-use images, use a container image based on the `eclipse/che-theia-endpoint-runtime:next` image that contains the dependencies.

> .Example: 

Reviewer: ```suggestion
.Example
```

> The `FROM` directive must correspond to `FROM eclipse/che-theia-endpoint-runtime:next`. This is required since the base image contains tools for running the remote VS Code extension and communicating between the sidecar and the Che-Theia editor. This way, the VS Code extension does not have to know that it is a remote one.

Reviewer: This changes meaning.
```suggestion
Base the `FROM` directive on `FROM eclipse/che-theia-endpoint-runtime:next`. This is required because the base image contains tools for running the remote VS Code extension and for communicating between the sidecar and the Che-Theia editor. This way, the VS Code extension operates as if it was not remote.
```
